sunscreen_runtime/src/linked.rs

```diff
@@ -411,18 +401,29 @@ impl SealSdlpProverKnowledge {
     /// Get the verifier knowledge component.
     ///
     /// Delegation to [`LogProofProverKnowledge::vk`].
-    pub fn vk(&self) -> SealSdlpVerifierKnowledge {
+    pub fn vk(&self) -> SdlpVerifierKnowledge {
         seq_zq!({
             match &self.0 {
                 #(
-                    SealSdlpProverKnowledgeInternal::LP~N(pk) => {
-                        SealSdlpVerifierKnowledge(SealSdlpVerifierKnowledgeInternal::LP~N(pk.vk.clone()))
+                    SdlpProverKnowledgeInternal::LP~N(pk) => {
+                        SdlpVerifierKnowledge(SdlpVerifierKnowledgeInternal::LP~N(pk.vk.clone()))
                     }
                 )*
             }
         })
     }
 
+    /// Get a mutable reference to the bounds on the secret `S`.
+    pub fn bounds_mut(&mut self) -> &mut Matrix<Bounds> {
+        seq_zq!({
+            match &mut self.0 {
+                #(
+                    SdlpProverKnowledgeInternal::LP~N(pk) => &mut pk.vk.bounds,
+                )*
+            }
+        })
+    }
```

Comment:
This allows people to set custom bounds when generating the prover knowledge. It's used by the builder, it could be made `pub(crate)`. I thought I had a lower-level method for people to get back an `SdlpProverKnowledge` but, looking again, there isn't, there's only a way to create the full `Sdlp`. So yea I can update this to be `pub(crate)`.